docs/source/interfaces/ISIS SANS v2.rst

```diff
@@ -513,6 +513,22 @@ Wavelength
 
 The settings provide the binning for the conversion from
 time-of-flight units to wavelength units. Note that all units are Angstrom.
+Depending on which Step type you have chosen you will be asked to enter either
+a Max and Min wavelength value between which to do the reduction or to specify a
+set of wavelength ranges to reduce between. The syntax for the latter case is the
+same as that used to specify event slices and is 
+
+- *start:step:stop* specifies wavelength slices from a *start* value for the *stop* value
```

Comment:
Could these be put in code markup (` `` `) instead of italics? Personally, I find this slightly more readable

Reply:
``